Mathlib/Analysis/Normed/Group/Uniform.lean

```diff
@@ -379,4 +379,16 @@ theorem cauchySeq_prod_of_eventually_eq {u v : ℕ → E} {N : ℕ} (huv : ∀ n
   intro m hm
   simp [huv m (le_of_lt hm)]
 
+@[to_additive CauchySeq.norm_bddAbove]
+lemma CauchySeq.norm_bddAbove' {G : Type*} [SeminormedCommGroup G] {u : ℕ → G}
```

Comment:
Any possibility of a better name? `mul_norm_bddAbove` perhaps?

(Perhaps we need to institute a rule that theorems ending in primes need a doc-string containing the explanation for why nothing better is available. :-)